modules/ztp-sno-du-subscribing-to-the-operators-needed-for-platform-configuration.adoc

```diff
@@ -13,4 +13,32 @@
 * PTP Operator
 * SR-IOV Network Operator
 
-include::snippets/ztp-operator-subs.adoc[]
+For each Operator subscription, specify the channel to get the Operator from. The recommended channel is `stable`.
+
+You can specify `Manual` or `Automatic` updates.
+In `Automatic` mode, the Operator automatically updates to the latest versions in the channel as they become available in the registry.
+In `Manual` mode, new Operator versions are installed only when they are explicitly approved.
```

Comment:
The recommended solution is to use Manual mode for subscriptions. This allows you to control the timing of Operator updates to fit within planned/scheduled maintenance windows.